src/structures/APIMessage.js

```diff
@@ -320,7 +320,7 @@ class APIMessage {
    * @returns {MessageOptions|WebhookMessageOptions}
    */
   static create(target, options, extra = {}) {
-    if (typeof options === 'string') return new this(target, { content: options, ...extra });
+    if (typeof options !== 'object' || options === null) return new this(target, { content: options, ...extra });
     else return new this(target, { ...options, ...extra });
```

Comment:
What about something like this? I'm unsure about the format Prettier will pick, so this might cause a format error:
```suggestion
    return new this(target, typeof options === 'object' || options !== null ?
    	{ ...options, ...extra } :
    	{ content: options, ...extra });
```